src/labelle/lib/constants.py

```diff
@@ -52,36 +53,110 @@ class SUPPORTED_DEVICE_ID(Enum):
     MOBILE_LABELER = 0x1009
 
 
-# fmt: off
-# Very bad I know, but it keeps a loop of removing the line breaks and errroring
+# Create supported products list
+SUPPORTED_PRODUCTS = []
 
-SUPPORTED_PRODUCTS = {
-    SUPPORTED_DEVICE_ID.LABELMANAGER_PC:
+# ---- Supported USB Devices configuration ----
+SUPPORTED_PRODUCTS.append(
+    DeviceConfig(
         "DYMO LabelMANAGER PC",
-    SUPPORTED_DEVICE_ID.LABELPOINT_350:
+        [int(SUPPORTED_DEVICE_ID.LABELMANAGER_PC)],
+        # ToDo: Validate config!
+        128,
```

Comment:
It's hard to understand from reading this what each field means. Please explicitly name each field.

Reply:
To be explicit, this is as simple as

```python
DeviceConfig(
    name="DYMO LabelMANAGER PC",
    deviceIDs=[int(SUPPORTED_DEVICE_ID.LABELMANAGER_PC)]
    ...
```

Also, the use here of `SUPPORTED_DEVICE_ID` seems very indirect to me. Why not just do

```
    deviceIDs=[0x0011]
```

Reply:
If that is possible can you give me a syntax example? (It would definitely be a lot clearer)

Reply:
@FaBjE that was an explicit example, and should work without other modifications to your code if you continue the obvious pattern. (Python lets you specify the names of positional arguments in the function invocation.) Does this make sense?

Reply:
Here's an example:

```python
def f(x, y):
    return 2 * x + y

f(y=1, x=2)  # 5
```

Reply:
Fixed